Source/HBIOS/simrtc.asm

```diff
@@ -55,9 +55,7 @@ SIMRTC_DISPATCH:
 	JP	Z,SIMRTC_SETALM	; SET ALARM 
 	DEC	A
 	JP	Z,SIMRTC_DEVICE	; REPORT RTC DEVICE INFO
-	CALL	SYSCHK
-	LD	A,ERR_NOFUNC
-	OR	A
+	SYSCHKERR(ERR_NOTFUNC)

```

Comment:
Please fix: ERR_NOTFUNC --> ERR_NOFUNC

Reply:
Thanks for all this work Phil.  I marked one small issue that causes a build error.  Can you fix this before I merge?

Thanks,

Wayne